ArduCopter/Parameters.cpp

```diff
@@ -172,8 +172,8 @@ const AP_Param::Info Copter::var_info[] = {
 
     // @Param: FS_GCS_ENABLE
     // @DisplayName: Ground Station Failsafe Enable
-    // @Description: Controls whether failsafe will be invoked (and what action to take) when connection with Ground station is lost for at least 5 seconds. NB. The GCS Failsafe is only active when RC_OVERRIDE is being used to control the vehicle.
-    // @Values: 0:Disabled,1:Enabled always RTL,2:Enabled Continue with Mission in Auto Mode (Deprecated in 4.0+),3:Enabled always SmartRTL or RTL,4:Enabled always SmartRTL or Land,5:Enabled always land (4.0+ Only)
+    // @Description: Controls whether failsafe will be invoked (and what action to take) when connection with Ground station is lost for at least 5 seconds. See FS_OPTIONS param for additional actions, or for cases allowing Mission continuation, when GCS failsafe is enabled.
+    // @Values: 0:Disabled/NoAction,1:RTL,2:RTL or Continue with Mission in Auto Mode (Deprecated in 4.0+-same as bit 1),3:SmartRTL or RTL,4:SmartRTL or Land,5:Land (4.0+ Only)
```

Comment:
Looking OK but I don't think that most users can do bit math... so I think we should just say "See FS_OPTIONS"

Reply:
what about 2:RTL or Continue with Mission in Auto Mode ( in 4.0+- just RTL)

Reply:
I think we should try and get users to stop using "2" though on 4.0+.  I think it should be:

2: RTL or Continue with Mission in Auto Mode (Deprecated in 4.0+ See FS_OPTIONS)